app/uk/gov/hmrc/overseaspensiontransferbackend/controllers/GetTransferDataController.scala

```diff
@@ -35,15 +36,16 @@ class GetTransferDataController @Inject() (cc: ControllerComponents, submissionS
       referenceId: String,
       pstr: String,
       qtStatus: String,
-      formBundleNumber: Option[String] = None,
-      qtNumber: Option[String]         = None,
-      versionNumber: Option[String]    = None
+      versionNumber: Option[String]
     ): Action[AnyContent] =
     Action.async {
       implicit request =>
         implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromRequest(request)
 
-        submissionService.getTransfer(referenceId, pstr, QtStatus(qtStatus), formBundleNumber, qtNumber, versionNumber) map {
+        val transferType: Either[TransferRetrievalError, GetSpecificTransferDTO] =
```

Comment:
This makes sense but it's not really a "DTO" so to speak as it's not transferred between the frontend and middleware, we could call it a helper function or something?